src/main/resources/config/application.yml

```diff
@@ -190,4 +192,5 @@ application:
     cassandra:
         migration-folder: config/cql/changelog
         keyspace-create-cql: "CREATE KEYSPACE IF NOT EXISTS %s WITH REPLICATION = {'class':'SimpleStrategy','replication_factor':3} AND DURABLE_WRITES = true;"
+        keyspace-to-use: "USE %s"
```

Comment:
@vkirichenko sugest to rename property to `keyspace-switch-cql`

Reply:
done